src/generic/stage2/numberparsing.h

```diff
@@ -418,7 +371,7 @@ simdjson_really_inline error_code write_float(const uint8_t *const src, bool neg
   // NOTE: it's weird that the simdjson_unlikely() only wraps half the if, but it seems to get slower any other
   // way we've tried: https://github.com/simdjson/simdjson/pull/990#discussion_r448497331
   // To future reader: we'd love if someone found a better way, or at least could explain this result!
-  if (simdjson_unlikely(exponent < FASTFLOAT_SMALLEST_POWER) || (exponent > FASTFLOAT_LARGEST_POWER)) {
+  if (simdjson_unlikely(exponent < smallest_power) || (exponent > largest_power)) {
```

Comment:
It's worth talking about casing and constants for style in the future. We don't have to resolve it here, though.

I find it very useful for constants to look different from variables. Variables make my brain go "OK I need to track this down, see where it comes from and range of values it can have." Knowing something is a constant often allows me to ignore it and focus on more important things.